lib/stringReplaceMiddleware.js

```diff
@@ -1,10 +1,14 @@
 const hijackResponse = require('hijackresponse');
 const stringReplaceStream = require('./stringReplaceStream');
 
-module.exports = (replacements) => (req, res, next) => {
+const defaultOptions = {
+  contentTypeFilterRegexp: /^text\/|^application\/json$|^application\/xml$/,
+}
+
+module.exports = (replacements, options = defaultOptions ) => (req, res, next) => {
     hijackResponse(res, function (err, res) {
         const contentType = res.get('content-type') || "";
-        if (contentType.startsWith('text')) {
+        if (contentType.match(options.contentTypeFilterRegexp)) {
```

Comment:
Better to do
```javascript
if (options.contentTypeFilterRegexp.test(contentType)) {
```

Reply:
Nice! I didn't like that I needed a default value of `""` for contentType. I made the change you suggested and removed the default value.